gerbera-web/test/client/fixtures/index.html

```diff
@@ -55,6 +58,11 @@ <h4><i class="fa fa-plus"></i> Add some files</h4>
                 <h4><i class="fa fa-files-o"></i> View indexed files</h4>
                 <p>View, update and remove files from Gerbera.</p>
             </a>
+
+            <a id="clientlink" href="javascript:;" onclick="$('#nav-clients').click(); // Replace with hashchange handling">
```

Comment:
I guess you are working on this still?

Reply:
Copy and paste, It's been in the source as well